web/components/application/hacker/pages/PageThree/PageThree.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+
+import { HackerApplicationPageTemplate } from '../PageTemplate';
+import { Page } from '../../../Page';
+import { constraints } from './PageThreeConstraints';
+import { Select } from '../../../../input/select';
+import { Checkbox, CheckboxGroup } from '../../../../input/buttons/CheckboxGroup';
+import { sources } from '../../../sources';
+
+// Emojis
+import robot from '../../../../../assets/emoji/robot.svg';
+import floppy from '../../../../../assets/emoji/floppy.svg';
+import briefcase from '../../../../../assets/emoji/briefcase.svg';
+
+class PageThree extends Page {
+  constructor(props) {
+    super(props);
+    this.constraints = constraints;
+  }
+
+  render() {
+    const {
+      application: {
+        source,
+        isPrivacyPolicyChecked,
+        isCodeOfConductChecked,
+        isDataReportingChecked,
+        isDocumentsChecked,
+      },
+    } = this.props;
+
+    return (
+      <HackerApplicationPageTemplate {...this.getPageTemplateProps()}>
+        <h2>Almost there...</h2>
+
+        <Select
+          options={sources}
+          value={source}
+          name="source"
+          label="How did you hear about nwHacks?"
+          placeholder="Select an option"
+          className="margin-ends-giga"
```

Comment:
Super minor but this should be `margin-top-mega margin-bottom-xxl` to match the mocks. Also you'll have to define `margin-top-mega` in `spacing.sass` as it's not defined yet: https://github.com/nwplus/nwhacks2019/blob/master/web/styles/spacing.sass#L144-L158

Reply:
Okay cool, didn't realize I could figure out exact pixel distances in Figma, fixed

Reply:
Ah yeah you can click anything in Figma then press alt and hover over something else to see the pixel distance